### DISCUSSION TITLE
Marketplace Feedback: VS Copilot unresponsive for every document.

### DISCUSSION THREAD

**Discussion by ayim** [community/community#50500] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Per Pedro Dias on VS Marketplace;
![image](https://user-images.githubusercontent.com/10951869/226368550-c28b545a-68fa-45f1-8275-679b16452aa0.png)

Copilot extension version unknown.

**Comment by paolomissagia** [comment]
I just found this post on the review section. I am experiencing the exactly same issue. Any file that I open, VS 2022 becomes unresponsive and the only way out is to force close the application.

I will post more details tomorrow once I am back again on my machine.

**Reply by ayim** [reply]
@paolomissagia it's hard to get a good repro sometimes when working with AI especially.
If you are open to it, we would love to do a screenshare/live debug with you.

Please shoot me an email at ayim at github dot com and we'll get the ball rolling.

**Reply by paolomissagia** [reply]
Hi @ayim. Sure, let's do it. I will drop you an email today.

**Reply by paolomissagia** [reply]
I've just sent you an email @ayim. As mentioned yesterday on my comment, here are the extra information.
GitHub Copilot version 1.77.1.0
Visual Studio Professional 2022 version 17.5.2

**Comment by FranciscoBucioDTI** [comment]
I have the same issue. I'm using Visual Studio 2022 Enterprise version 17.5.3 and Copilot 1.77.2.0 and VS2022 becomes unresponsive when opening a file.

However, I've been able to find a workaround, maybe it helps you finding the error.

* When I open VS, nothing shows under "Output window", as if GitHub Copilot service hasn't started yet:
![image](https://user-images.githubusercontent.com/110624108/226866399-e510b613-4b4f-4706-87c5-2688c4468e64.png)

* The moment I select "Github" at Tools - Options, "Output window" displays info about GitHub Copilot service starting:
![image](https://user-images.githubusercontent.com/110624108/226868923-d8215358-5aeb-4f27-8ea9-c4fbbcf4e02f.png)

After that, I have no problem opening any file, it doesn't freezes anymore.

**Reply by paolomissagia** [reply]
I can confirm the above solution. This workaround has fixed the issue for me as well. @ayim 

Thanks for that @FranciscoBucioSTI

**Comment by dcarapic** [comment]
@FranciscoBucioSTI Do you have to do this every time you open visual studio?

**Reply by FranciscoBucioDTI** [reply]
Yes. Every time I have to do this, otherwise VS freezes

**Comment by digitaldias** [comment]
I was the original poster, and I confirmed this is still a problem. Also happy to say that the workaround ALSO works:

### Workaround
* Before opening any project/solution so 
* Open up Options -- GitHub 
* Verify the output window now shows initialization from GitHub Copilot
* Now you can open up your project/solution 

This has to be done every time you open up a new instance of Visual Studio.

### Versions
**Visual Studio Version Enterprise**: 17.6.0 Preview 2.0
**GitHub Copilot Extension**: 1.77.2.0

**Comment by digitaldias** [comment]
I downloaded 1.79.0.0 and the problem is now gone on my part. Not sure if I mark it as resolved here :)
